stable/socat-tunneller/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "tunneller.fullname" . }}
+  labels:
+    app: {{ include "tunneller.name" . }}
+    chart: {{ include "tunneller.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
Can you please use the common label names shared by Kubernetes. See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata